python/cudf/cudf/core/dataframe.py

```diff
@@ -3955,6 +3955,12 @@ def apply(
         For more information, see the `cuDF guide to user defined functions
         <https://docs.rapids.ai/api/cudf/stable/user_guide/guide-to-udfs.html>`__.
 
+        Support for use of string data within UDFs is provided through the
+        `strings_udf <https://anaconda.org/rapidsai-nightly/strings_udf>`__ RAPIDS
```

Comment:
Sorry, I wasn't paying attention to flake8's annoyingly short line lengths. This paragraph needs to be rewrapped I think.